layouts/partials/article-meta/basic.html

```diff
@@ -47,7 +47,7 @@
 {{ end }}
 
 
-<div class="flex flex-row flex-wrap items-center">
+<div style="cursor: default;" class="flex flex-row flex-wrap items-center">
```

Comment:
@nunocoracao I think this hides the pointer cursor for article date link. Is that on purpose?

Reply:
@JVimes yes, did you have any use for it?

Reply:
Without it, I'd assume the site doesn't link article dates.

Reply:
Sorry @JVimes I am not following the use-case. Can you point me to an example of what worked and doesn't anymore. Maybe we should open a issue for this also.

Reply:
Never mind, I'm getting my wires crossed with my modifications 🙂